ts/components/RemindEmailValidationOverlay.tsx

```diff
@@ -310,26 +309,24 @@ class RemindEmailValidationOverlay extends React.PureComponent<Props, State> {
                 <Text>{this.state.ctaSendEmailValidationText}</Text>
               </Button>
             )}
+          {this.state.displayError && (
```

Comment:
Could it be a toast (the red one) instead of a Text?
<img src="https://user-images.githubusercontent.com/822471/77459298-ce182b80-6dff-11ea-9eb6-eb5122cc1106.png" height="600"/>

Reply:
updated as describer below ;)